docs/src/staking/stake-accounts.md

```diff
@@ -0,0 +1,106 @@
+## Stake Account Structure
+A stake account on the Solana can be used to delegate tokens to validators on
+the network to potentially earn rewards for the owner of the stake account.
+Stake accounts are created and managed differently than a traditional wallet
+address, also known as a system account,
```

Comment:
That might be more digestible if split into two sentences. "different than a traditional wallet address, known as a system account. A system account can..., whereas a stake account..."